packages/components/Callout/package.json

```diff
@@ -29,14 +29,14 @@
     "@uifabricshared/foundation-settings": ">=0.8.0 <1.0.0"
   },
   "devDependencies": {
-    "@types/react-native": "^0.62.0",
+    "@types/react-native": "^0.63.0",
     "@uifabricshared/build-native": "^0.1.1",
     "@uifabricshared/eslint-config-rules": "^0.1.1",
-    "react-native": "0.62.2"
+    "react-native": "^0.63.4"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "react-native": ">=0.60.0"
+    "react": ">=16.13.1",
+    "react-native": ">=0.63.4"
```

Comment:
General question for Microsoft packages/components, do we prefer to keep the minimum 0.62 since that technically would still work, or to keep the minimum the highest version of RN as possible?

Reply:
0.62 would probably at least mostly work.  But for instance we are now setting the Focusable property (0.63 supported on all platforms), rather than acceptsKeyboardFocus (0.62 only on windows/macos and our office RN fork).   We could try to support 0.62 also, but then we'd need some weird casts to allow us to set both properties, and ensure that we do set both when someone specifies one of them.  And generally be quick a headache, that would be undertested.

As far as I know all our clients are moving to 0.63, so we should just reduce our testing / support matrix while we can.

Reply:
The point about the focusable property makes sense! That does set our minimum to 0.63 for just about everything, thanks!